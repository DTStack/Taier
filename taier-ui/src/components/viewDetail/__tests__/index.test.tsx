import { render } from '@testing-library/react';
import ViewDetail from '..';

describe('Test ViewDetail Component', () => {
    it('Should match snapshot', () => {
        const { asFragment } = render(<ViewDetail />);

        expect(asFragment()).toMatchSnapshot();
    });
});
