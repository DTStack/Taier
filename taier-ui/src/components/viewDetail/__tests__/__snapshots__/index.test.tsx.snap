// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ViewDetail Component Should match snapshot 1`] = `
<DocumentFragment>
  <p
    data-testid="antd-mock-Modal-title"
  >
    任务详情
  </p>
  <div
    class="code-editor"
    style="position: relative; min-height: 400px; height: 400px; width: 100%; margin-top: 1px;"
  >
    <pre
      class="dt-placeholder"
      style="font-size: 12px; display: initial;"
    />
  </div>
  <button
    data-testid="antd-mock-Modal-confirm"
  >
    confirm
  </button>
</DocumentFragment>
`;
